Automattic/wp-calypso
# Add Boost column to Pro dashboard

Related to #1203940061556608-as-1204003850818239/f

## Proposed Changes

This PR implements the Boost column to the main Agency Dashboard

## Testing Instructions

* Run `add/boost-column-pro-dashboard` and `yarn start-jetpack-cloud`
* Open http://jetpack.cloud.localhost:3000/, and you'll be redirected to the /dashboard.
* You'll see in the Boost column, the default values for Boost (right now, it's 100 for every site - so will display an *A Score*)

![image](https://user-images.githubusercontent.com/2293077/224160136-a4b8f73a-4a06-473b-a78a-00ac5d62aad9.png)

![image](https://user-images.githubusercontent.com/2293077/223600689-d856810b-71a4-4ad8-aa7f-78ba2b0df0db.png)


To test for different values, comment out line 183 in `site-status-content.tsx` and create a new variable `overall` and set it equal to different number values.

The ranges that should be triggered can be found here:

<img src="https://user-images.githubusercontent.com/2293077/223558788-8572b606-c0cb-42c4-a99d-4b4ec8f58760.png" width="200">

Scores for *A and B* are green, *C and D* are yellow, and *E and F* are red.

## Notes

- If a site does not have Boost enabled, then the cell will be blank (will add the +Add button in a separate PR)

## Pre-merge Checklist
